lib/mtif/posts.rb

```diff
@@ -125,20 +125,26 @@ def mtif_value(value)
       value.kind_of?(Time) ? value.strftime(DATE_FORMAT) : value
     end
 
-    def convert_to_native_type(raw_value)
+    def convert_to_native_type(raw_value, key)
+      if MULTILINE_KEYS.include?(key)
+        return raw_value
+      end
+
+      if key == :date && raw_value.match(/^\d{2}\/\d{2}\/\d{4} \d{2}:\d{2}:\d{2} [AP]M/)
```

Comment:
From the look of their official spec https://movabletype.org/documentation/appendices/import-export-format.html I think `date` is the only one of its kind.